internal/rbd/driver.go

```diff
@@ -199,5 +199,14 @@ func (r *Driver) Run(conf *util.Config) {
 		util.DebugLogMsg("Registering profiling handler")
 		go util.EnableProfiling()
 	}
+	if conf.IsNodeServer {
+		go func() {
+			// TODO: move the healer to csi-addons
```

Comment:
@pkalever may be have a tracker issue for this TODO

Reply:
Here: https://github.com/csi-addons/spec/issues/20